ui/app/styles/core/forms.scss

```diff
@@ -189,16 +189,18 @@ label {
 }
 
 .select:not(.is-multiple)::after {
-  border-color: $black;
-  border-width: 2px;
-  margin-top: 0;
-  transform: translateY(30%) rotate(-45deg);
-}
-
-.select:not(.is-multiple)::before {
-  content: '';
-  transform: translateY(-70%) rotate(135deg);
-  z-index: 5;
+  background-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjQiIGhlaWd\
```

Comment:
You could use the `svg-uri` helper here so that the SVG code is still readable - here's an example of it: 
https://github.com/hashicorp/vault/blob/8f481125f74e9e44bd32a896a249fdb76dfc13bf/ui/app/styles/core/bulma-radio-checkboxes.scss#L11

Reply:
That's amazing. Updated.